subprojects/studio-plugin/src/main/java/org/gradle/profiler/studio/plugin/GradleProfilerProjectManagerListener.java

```diff
@@ -0,0 +1,19 @@
+package org.gradle.profiler.studio.plugin;
+
+import com.intellij.ide.impl.TrustedProjects;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.project.ProjectManagerListener;
+import org.gradle.profiler.studio.plugin.client.GradleProfilerClient;
+
+public class GradleProfilerProjectManagerListener implements ProjectManagerListener {
+
+    private static final Logger LOG = Logger.getInstance(GradleProfilerProjectManagerListener.class);
+
+    @Override
+    public void projectOpened(com.intellij.openapi.project.Project project) {
+        LOG.info("Project opened");
+        TrustedProjects.setTrusted(project, true);
```

Comment:
You are setting trusted here, and you also pass the system property to trust all projects. Can we remove one of the two things?

Reply:
Hawk eyes. Yes, I will remove the property. The thing is that property was added in one of the latest IntelliJ, but is not supported for example in 2021.1 what is used by Android Studio Bumblebee atm.

Reply:
It is also fine to leave both things, as long as there is a comment somewhere.